src/steps/project/converter.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  createIntegrationEntity,
+  Entity,
+} from '@jupiterone/integration-sdk-core';
+
+import { Entities } from '../constants';
+import { CircleciProject } from '../../types';
+
+export function getProjectKey(id: string): string {
+  return `circleci_project:${id.toString()}`;
```

Comment:
It is a string already.

```suggestion
  return `circleci_project:${id}`;
```